docker-compose-mon.yml

```diff
@@ -0,0 +1,151 @@
+version: "3"
+
+services:
+  db:
+    image: postgres:15
+    restart: always
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    environment:
```

Comment:
Instead of duplicating configurations for each service, could you extend from `docker-compose.yml`.
Example:
https://github.com/filecoin-project/motion/blob/51e4d293357d6d3eee38236c827d307e213de45c/docker-compose-local-dev.yml#L9